src/Extension/AsyncCMSMain.php

```diff
@@ -93,17 +93,14 @@ public function asyncSave(array $data, Form $form): HTTPResponse
      */
     public function asyncGetRecordAndAssertPermissions(array $data)
     {
-        $className = $this->owner->config()->get('tree_class');
-        if (!$className) {
-            $className = $this->owner->config()->get('model_class');
-        }
+        $className = $this->owner->config()->get('model_class');
 
         // Existing or new record?
         $id = $data['ID'];
 
         if (!str_starts_with($id ?? '', 'new')) {
             /** @var SiteTree $record */
-            $record = DataObject::get_by_id($className, $id);
+            $record = DataObject::get($className)->setUseCache(true)->byID($id);
```

Comment:
`setUseCache()` was introduced in `6.1.0.` - if we want to use this, lets bump the min requirement to CMS 6.1, otherwise it'll go boom if anyone is still using 6.0.x

Reply:
I reckon bump the composer requirement, saves us double-handling. Devs on 6.0.x will still have the option of the previous release

Reply:
Done